pkg/ansible/watches/watches_test.go

```diff
@@ -137,8 +143,8 @@ func TestLoad(t *testing.T) {
 				Kind:    "NoFinalizer",
 			},
 			Playbook:                    validTemplate.ValidPlaybook,
-			ManageStatus:                true,
-			ReconcilePeriod:             twoSeconds,
+			ManageStatus:                ptrBool(true),
```

Comment:
Why the `ManageStatus` needs to be `*bool` and not `bool`? 
However, instead of `ptrBool` is possible just
manageStatus := true and the here &manageStatus

Reply:
Because many of the test data in the **valid.yaml.tmpl** have the ManageStatus as empty field. If it is of type `bool`, then the default value in tmp (object of alias struct) is getting set as false. This is making the ManageStatus as false in watch struct as well. So I have made it a `*bool`. Now if that field is empty in the test file, then the `*bool` will have a `nil`. In that case we can set it as true. If not, we set the value it has. We need that tertiary condition where bool offers only two.